### ISSUE TITLE
(Gearswap)Instant casting not always swapping into midcast as it's suppose to.

### ISSUE THREAD

**Issue by Sevenz-XI** [Windower/Lua#1555]
Hi, for about a month now I have been having issues with instant casts on my BRD occasionally not swapping into midcast gear. I suspect this is happening with all my jobs and it's just most obvious on BRD.

Firstly the dead give away, duration is gimp. Then I took a moment to test it and when I get a gimp song the potency is as if I casted with +0

It has become extremely annoying to have to lock slots to ensure my songs dont get gimped. It's annoying not being able to use nitro or pianissimo and to essentially play the game vanilla with a half functioning addon...

Also I suspect that my main probably is not always swapping into WS gear or JA gear on DD jobs etc.

Something is clearly wrong with the precast to midcast functionality, I spoke with a few others who have noticed similar issues.

I can post my BRD lua but I can ensure you it is not the job file.

**Comment by cairthenn** [comment]
Hey @Byrth , I know you are aware this happens sometimes, particularly in laggy instanced areas, but do we know exactly what's happening? Do the chunks just get sent in the wrong order, are some just not sent?

I have an idea of what may be the cause, but I'm not sure exactly how the issue manifests.

**Comment by KenshiDRK** [comment]
This has been discussed in the ffxiah forums http://www.ffxiah.com/forum/topic/50799/gearswap-problems/ and http://www.ffxiah.com/forum/topic/50486/gearswap-v0927-inventory-handling-changes/

**Comment by Sevenz-XI** [comment]
Well it's good to hear some explanation about this. Also the server population being a factor absolutely must be part of it. I was on a very small server not long before I started noticing these persistent issues which was when I transferred to one of the highest populated servers. Seems like just another reason to transfer again...

However if Atam0s and lolwat have this figured out and resolved for ashitacast I do not see why this can't be fixed. Almost nobody wants to use ashitacast and basically revert back to spellcast after getting a taste of gearswap.

I wish I could help but I'm really only knowledgeable on the user end of gearswap and have no programing experience.

@Byrth Will we be possibly seeing some revision done to Gearswap to resolve this issue?
